datasets/ph/most_wanted/crawler.py

```diff
@@ -0,0 +1,36 @@
+import csv
+from typing import Dict
+import zavod.helpers as h
+from zavod import Context
+
+
+def crawl_row(context: Context, row: Dict[str, str]):
+    data = dict(row)
+    full_name = data.pop("name", None)
+    offense = data.pop("offense", None)
+    case_number = data.pop("case number", None)
+    source = data.pop("source", None)
+    reward = data.pop("reward", None)
+
+    entity = context.make("Person")
+    entity.id = context.make_id(full_name, case_number, reward)
+    entity.add("name", full_name)
+    entity.add("sourceUrl", source)
```

Comment:
we usually only add the source url when it's specific to the person, or when the data has been extraced from a document from a collection like a gazette or regulation listing a set of items. In this case the user can find this link via the dataset, so you can remove.